addons/common/functions/fnc_arithmeticGetResult.sqf

```diff
@@ -18,51 +18,44 @@
  * Public: Yes
  */
 
-params ["_namespace", "_setID", "_op"];
-TRACE_3("arithmeticGetResult",_namespace,_setID,_op);
+params ["_namespace", "_setID", "_operation"];
+TRACE_3("arithmeticGetResult",_namespace,_setID,_operation);
 
 private _data = (_namespace getVariable _setID) param [2, []];
```

Comment:
I do NOT think this is necessary, but it would make the func safer as `selectMax []` is undefined (and div0 in "avg")
```suggestion
private _data = (_namespace getVariable _setID) param [2, [{0}]];
```